packages/vuetify/src/components/VDataTable/VDataTable.sass

```diff
@@ -42,6 +42,14 @@
             &.v-data-table-column--no-padding
               padding: 0 8px
 
+            &.v-data-table-column--wrap
```

Comment:
It seems confusing that the name of the class is --wrap but it removes wrap instead.

Reply:
Do I need to change class name to nowrap? and prop to "nowrap"